devutils/pyproject.toml

```diff
@@ -28,6 +28,9 @@ tomlkit = "^0.11.8"
 semver = "^3.0.0"
 pylint = "^2.17.4"
 
+[tool.poetry.group.dev.dependencies]
+mdformat = "^0.7.17"
```

Comment:
This was required when I ran for the first time `inv docs` in **storage**. (placed here in case it is required by other packages as well)

```bash
% inv docs
poetry run lazydocs --validate --no-watermark --remove-package-prefix --src-base-url https://github.com/robocorp/robo/tree/master/ --overview-file README.md --output-path /Users/cmin/Repos/robo/storage/docs/api robocorp.storage
Traceback (most recent call last):
  File "/Users/cmin/Repos/robo/storage/.venv/bin/lazydocs", line 5, in <module>
    from lazydocs._cli import app
  File "/Users/cmin/Repos/robo/storage/.venv/lib/python3.9/site-packages/lazydocs/__init__.py", line 6, in <module>
    from .generation import MarkdownGenerator, generate_docs
  File "/Users/cmin/Repos/robo/storage/.venv/lib/python3.9/site-packages/lazydocs/generation.py", line 18, in <module>
    import mdformat
ModuleNotFoundError: No module named 'mdformat'
```

Reply:
But then this shouldn't be here... Rather it should be in the `devutils` `pyproject.toml`.

Reply:
Hmm, I moved it actually to [devutils/pyproject.toml](https://github.com/robocorp/robo/pull/90/files/3ab309cdcc31b476011b3007a740388125eed50b#diff-74077cadcb57aa19e6ddb0eb0cf03cd645d31154a89061abb758130015c76f97) isn't that? (initially I placed it in **storage**'s one, then refactored before push)

Reply:
Seems I saw it wrong there, sorry for the noise.